js_modules/dagit/packages/core/src/runs/LogsProvider.tsx

```diff
@@ -161,6 +167,20 @@ const useLogsProviderWithSubscription = (runId: string) => {
     },
   });
 
+  // If the websocket reconnects, ensure the newly recreated subscription resumes from
+  // after the messages we have already loaded
+  React.useEffect(() => {
+    if (!websocketClient) {
+      return;
+    }
+    const unlisten = websocketClient.onReconnected(() => {
```

Comment:
should this be `onReconnecting` (or `onDisconnected`)?  just not obvious what the lifecycle is between the subscription request and the websocket client.

Reply:
ya good call - worth experimenting with that